src/avrt.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright © 2016, Baptiste AUBRY
+// Licensed under the MIT License <LICENSE.md>
+
+ENUM_NEG!{enum AVRT_PRIORITY {
+    AVRT_PRIORITY_VERYLOW = -2,
+    AVRT_PRIORITY_LOW,
+    AVRT_PRIORITY_NORMAL,
+    AVRT_PRIORITY_HIGH,
+    AVRT_PRIORITY_CRITICAL,
+}}
+
+ENUM!{enum AVRT_NOTIFICATION {
```

Comment:
Strangely enough, `AVRT_NOTIFICATION` was actually removed in the Windows 10 SDK. Since I try to track the latest Windows 10 SDK with winapi, I'd rather this not be here, unless you can provide a compelling reason for why it should be kept.